tests/operator/operator.go

```diff
@@ -117,13 +117,15 @@ type vmYamlDefinition struct {
 	vmSnapshots   []vmSnapshotDef
 }
 
+const (
+	imageDigestShaPrefix = "@sha256:"
```

Comment:
nit: It seems ensureShasums is the only user for this const, can we move it to there?

Reply:
This constant is also used in the `"[test_id:3144]using virt-launcher with a shasum"` test